pkg/descriptions.json

```diff
@@ -1042,6 +1042,15 @@
         "System.Buffers.Text.Utf8Formatter"
     ]
   },
+  {
+    "Name": "Microsoft.Compatibility.AsyncInterfaces",
+    "Description": "Provides compatibility classes for Async Interfaces",
```

Comment:
Could we say something more like "Provides interfaces and helper types for consuming and producing async enumerables on .NET Standard 2.0"?

Reply:
sure, although do we want to add the .NET Standard 2.0 on it as well on the description? I only ask because technically the package will contain the full facades for other frameworks as well (netstandard2.1, netcoreapp3.0, etc)

Reply:
I think it warrants to mention that the package is not required when targeting netstandard2.1 or netcoreapp3.0, unless consuming some other library which depends on this.